src/helpers/supportsModuleWorkers.ts

```diff
@@ -0,0 +1,18 @@
+export default function () {
+  let supports = false
+
+  const testerFunction = {
+    get type(): 'module' | 'classic' {
+      supports = true
+      return this.type
+    },
+  }
+
+  try {
+    const worker = new Worker('blob://;', testerFunction)
```

Comment:
`blob://` will save us from really creating a new worker, as stated [here](https://stackoverflow.com/a/62963963/14385332)

Reply:
You should write a comment in code about this one